pages/index.js

```diff
@@ -1,17 +1,22 @@
 import ActivityList from "@/components/ActivitiesList";
 import ActivityForm from "@/components/ActivityForm";
+import NavigationBar from "@/components/Navigation";
 import styled from "styled-components";
 
 export default function HomePage() {
   return (
-    <StyledMain>
-      <h1>Activity Planner</h1>
-      <ActivityForm />
-      <ActivityList />
-    </StyledMain>
+    <>
+      <StyledMain>
+        <h1>Activity Planner</h1>
+        <ActivityForm />
+        <ActivityList />
+      </StyledMain>
+      <NavigationBar />
```

Comment:
I would think that StyledMain h1 and navigationbar should be displayed in all pages. If that is the case, please create a Layout component, add Main, h1 and Navbar with {children} at the correct position. and wrap the <Component> with the <Layout> in the _app.js